pkgs/sdk/server/contract-tests/TestService.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\src\LaunchDarkly.ServerSdk\LaunchDarkly.ServerSdk.csproj" />
+    <ProjectReference Include="..\src\LaunchDarkly.ServerSdk.csproj" />
```

Comment:
For reviewers: Intentionally removed stuttery folder structure.

Reply:
During discussion with Ryan, we talked about reorganizing the directories to match the namespacing.  I thought about this more and chose not to due to a mixture of total time this story is taking and fear that release-please won't like the overlapping namespacing (I haven't dealt with release-please directory exclusions and don't really want to go down that rabbit hole at this moment).